app/Http/Transformers/PaginatorForPhoenixAshesGallery.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use League\Fractal\Serializer\ArraySerializer;
+use League\Fractal\Pagination\PaginatorInterface;
+
+class PaginatorForPhoenixAshesGallery extends ArraySerializer
+{
+    /**
+     * Serialize a collection.
+     *
+     * @param string $resourceKey
+     * @param array  $data
+     *
+     * @return array
+     */
+    public function collection($resourceKey, array $data)
+    {
+        return ['data' => $data];
+    }
+
+    /**
+     * Serialize the paginator.
+     *
+     * @param PaginatorInterface $paginator
+     *
+     * @return array
+     */
+    public function paginator(PaginatorInterface $paginator)
+    {
+        $currentPage = (int) $paginator->getCurrentPage();
+        $lastPage = (int) $paginator->getLastPage();
+
+        $pagination = [
+            'total' => (int) $paginator->getTotal(),
+            'count' => (int) $paginator->getCount(),
+            'per_page' => (int) $paginator->getPerPage(),
+            'current_page' => $currentPage,
+            'total_pages' => $lastPage,
+        ];
+
+        $pagination['links'] = [];
+
+        if ($currentPage > 1) {
+            $pagination['links']['previous_uri'] = $paginator->getUrl($currentPage - 1);
```

Comment:
me too - awesome suggestion and learned a lot to think outside of the box and override things instead of always playing by the rules :)